README.md

```diff
@@ -1,24 +1,55 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# BUDGETIFY
 
-Things you may want to cover:
+> A mobile web application where you can manage your budget: you have a list of transactions associated with a category,
+> so that you can see how much money you spent and on what.
 
-* Ruby version
+## Demo
+Check out the live app [here](https://limitless-castle-88719.herokuapp.com/)
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby on Rails
+- PostgresSQL
 
-* Database creation
+## Getting Started
 
-* Database initialization
+To get a local copy up and running follow these simple example steps.
 
-* How to run the test suite
+- Copy this link `https://github.com/kelvinbush/budgetify.git`.
+- Get the directory that you want to clone the repository.
+- Open the command prompt in this directory.
+- Write `git clone https://github.com/kelvinbush/budgetify.git`.
+- Go to the repository folder in your command prompt `cd budgetify`.
+- run `bundle install` to install dependencies
+- run `rails c` to start the local app.
+- run `rspec` to run the tests.
 
-* Services (job queues, cache servers, search engines, etc.)
+## Authors
 
-* Deployment instructions
+👤 **Kelvin Bush**
 
-* ...
+- GitHub: [@kelvinbush](https://github.com/kelvinbush)
+- Twitter: [@kelvin_bush_](https://twitter.com/kelvin_bush_)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kelvin-wachiye-04b469173/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Please fill the acknowledgments part of your readme. Because The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author.